Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -87,6 +88,10 @@ class RepositoryIssuesViewController: BaseListViewController<NSString>, BaseList
         return [searchKey]
     }
 
+    func models(listAdapter: ListAdapter) -> [ListDiffable] {
```

Comment:
Why/where do we use this?

Reply:
See new required function for `BaseListViewControllerDataSource`